src/lib/seam/components/CreateClimateSettingScheduleForm/CreateClimateSettingScheduleForm.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Dialog } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { CreateClimateSettingScheduleForm } from 'lib/seam/components/CreateClimateSettingScheduleForm/CreateClimateSettingScheduleForm.js'
+import { useToggle } from 'lib/ui/use-toggle.js'
+
+/**
+ * These stories showcase creating a climate setting schedule
+ */
+const meta: Meta<typeof CreateClimateSettingScheduleForm> = {
+  title: 'Example/CreateClimateSettingScheduleForm (Coming Soon)',
```

Comment:
Let's move it to the description, otherwise the links have to change when we remove it.
```suggestion
/**
 * Coming Soon
 * These stories showcase creating a climate setting schedule.
 */
const meta: Meta<typeof CreateClimateSettingScheduleForm> = {
  title: 'Components/CreateClimateSettingScheduleForm',
```